src/spikeinterface/core/generate.py

```diff
@@ -938,6 +938,7 @@ def generate_templates(
             * 'positive_amplitude': the positive amplitude in a.u. (default range: (0.05-0.15)) (negative is always -1)
             * 'smooth_ms': the gaussian smooth in ms (default range: (0.03-0.07))
             * 'decay_power': the decay power (default range: (1.2-1.8))
+            * 'propagation_speed': mimic a propagation delay with a kind of a "speed" (default range: (250., 350.)).
```

Comment:
Can't add a comment since you didn't edit in this PR, but the default for 'alpha' has also been changed so in line934 it should be changed to the new default (6_000.0,  9_000.0)